pxf/build.gradle

```diff
@@ -122,6 +124,47 @@ subprojects { subProject ->
 }
 
 project('pxf-service') {
+
+// Copy existing sources and replace any occurrences of @tokenName@ with desired values
+    task generateSources {
+        doFirst {
+            copy {
+                from('src/main/java') {
+                    include '**/*.java'
```

Comment:
This task just copies files from one place to another and replaces all occurrences of passed tokens, therefore we have to copy all *.java files to temp location.